fe/pom.xml

```diff
@@ -410,7 +421,8 @@ under the License.
             <dependency>
                 <groupId>io.netty</groupId>
                 <artifactId>netty-all</artifactId>
-                <version>4.1.61.Final</version>
+                <!--same with hadoop 3.4.0-->
+                <version>4.1.100.Final</version>
```

Comment:
This cannot be upgraded and will affect stream load

Reply:
fix stream load later